TicketDesk/ngWebClient-StandAlone/src/app/models/ticket-stub.ts

```diff
@@ -12,5 +12,14 @@ export interface TicketStub {
   }
 
 // Headings for table (displayed to User)
-export const columnHeadings: string[] = ['TicketID', 'Title', 'Status', 'Priority', 'Owner', 'Assigned To', 'Category',
-                                      'Created Date', 'Last Updated'];
+export const columnHeadings: string[] = [
+  'TicketID',
```

Comment:
my guess would be that this should be Ticket ID

Reply:
fixed